### PR TITLE
Move tour modal on short windows

### LINKED ISSUE #3340: Tour steps modal is squashed on iPad
**Describe the bug**
The tour functionality works fine on an iPad in landscape mode but, the modal with step info is squashed making the text hard to read.  This doesn't appear to happen on a desktop at the same resolution for some reason.

**Screenshots**
IMG_0322

 - Device: iPad AIr 2
 - OS: iOS
 - Browser: Safari

### PR DESCRIPTION
**Pull request by jasontk19**
## Description

Fixes #3340

On iPad height:
* moves the tour modal left and makes it taller
* makes all corners equally rounded
* reduces shadow a bit since the default looked heavy over the zoom/rotate buttons

### CODE REVIEW

**Comment by jasontk19** [comment]
Note that the iPad device view in Chrome devtools is inaccurately tall since the browser viewport contents take up the entire screen.  On a real device, the browser address bar and tabs take up quite a bit more vertical real estate.

**Comment by edplato** [comment]
I like the change moving the dialog to not cover up the map zoom buttons and that got me trying out some other changes too. I was getting toolbar buttons covered up on certain heights, so made changes below to fix that and keep buttons uncovered when resizing a desktop browser with short windows (for the most part - polar rotation buttons are still getting covered at certain heights) . Take a look and let me know what you think.

```
@media (max-height: 776px) {
  .tour-in-progress .modal {
    bottom: 290px;
  }
  .tour-in-progress .modal-content {
    height: 300px;
  }
}

@media (max-height: 720px) {
  .tour-in-progress .modal {
    right: 55px;
    .modal-dialog {
      box-shadow: 2px 2px 16px rgba(0, 0, 0, 0.45);
      .modal-content {
        border-radius: 8px;
      }
      .modal-header {
        border-top-right-radius: 8px;
        border-top-left-radius: 8px;
      }
      .modal-footer {
        border-bottom-right-radius: 8px;
        border-bottom-left-radius: 8px;
      }
    }
  }
}

@media (max-height: 550px) {
  .tour-in-progress .modal {
    bottom: calc(var(--vh, 1vh) * 100 - 265px);
  }
}

```

**Comment by jasontk19** [comment]
I'm not sure which selectors should get replaced (other than the first one) with the ones in your suggestion.  Do the `720px` and `550px` replace any existing selectors or are they entirely additional?

**Comment by edplato** [comment]
The `@media (max-height: 550px) {` is a new one, and the `media (max-height: 720px) {` was the `712px` breakpoint, but raised slightly so its changes kick in earlier and prevents the zoom out map button from any cover up.

**Comment by jasontk19** [comment]
@edplato Nice, looks great! Pushed those changes.

File: web/css/tour.css [hunk 1]
```diff
@@ -680,32 +681,40 @@
...
-    bottom: 178px;
-  }
-
-  .tour-in-progress .modal-content {
-    height: 200px;
+    bottom: calc(var(--vh, 1vh) * 100 - 265px);
   }
 }
 
+
```
Comment (edplato) [inline web/css/tour.css:717]:
```suggestion
```

**Review by edplato — APPROVED** [review]
Thanks, looks good. I made a comment to remove an extra line that was throwing a lint css warning.

**Pull request merged**